src/main/java/clickup/ui/pages/StarPage.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package clickup.ui.pages;
+
+import clickup.ui.BasePage;
+import core.selenium.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * StarPage Class.
+ *
+ * @author Jesus Menacho
+ * @version 1.0
+ */
+public class StarPage extends BasePage {
+    @FindBy(css = ".cu2-project-list-bar__add-icon > .ng-star-inserted")
+    private WebElement addNewButton;
+
+    @FindBy(css = ".cu-form__input")
+    private WebElement inputNameSpaceTextBox;
+
+    @FindBy(css = ".cu-btn")
+    private WebElement NextButton;
+
+    /**
+     * This Method create a new space.
+     *
+     * @param nameSpace String parameter.
+     */
+    public void addNewSpace(String nameSpace) {
+        addNewButton.click();
+        inputNameSpaceTextBox.sendKeys(nameSpace);
+        for (int index = 0; index < 7; index++) {
+            NextButton.click();
+        }
+    }
+
+    /**
+     * This method find name space.
+     *
+     * @param nameSpace parameter string.
+     * @return boolean result.
+     */
+    public boolean isFoundNameSpace(String nameSpace) {
+        return WebDriverManager.getInstance().getWebDriver().getPageSource().contains(nameSpace);
```

Comment:
You shouldn't be using WebDriverManager.
You may use driver directly, because this class is actually extending BasePage so driver is readily available.

Reply:
Done.